.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+Metrics/LineLength:
+  Max: 120
+Style/Documentation:
+  Enabled: false
+
+AllCops:
+  TargetRubyVersion: 2.2.1
+  Exclude:
+    - 'db/migrate/*'
```

Comment:
Let's add:
```yaml
  - 'lib/tasks/*'
```
This will ignore any rake tasks we add. Most rake tasks are auto-generated by gems (cucumber, etc.) so it doesn't make a lot of sense to mess with them. We should be able to handle the few we make ourselves.